pkg/debian/build.sh

```diff
@@ -68,10 +68,28 @@ fakeroot dpkg-deb --build "${SERVERROOT}" "${DISTROOT}/${APP_NAME}-server_${APP_
 echo "Creating the desktop package..."
 mkdir "${DESKTOPROOT}/DEBIAN"
 
-cat << EOF > "${DESKTOPROOT}/DEBIAN/conffiles"
+# Ubuntu 24 requires apparmor profile to work.
+UBUNTU_CODENAME=$(grep "^UBUNTU_CODENAME=" /etc/os-release | awk -F "=" '{ print $2 }')
+if [ "${UBUNTU_CODENAME}" == 'noble' ]; then
```

Comment:
Hard coded on a single codename? That means we need to change the code with every release. Please change to a numeric comparison on the version number.